common/models/load_balancer_type.go

```diff
@@ -0,0 +1,41 @@
+package models
+
+import (
+	"strings"
+)
+
+type LoadBalancerType string
+
+const (
```

Comment:
This seems like overkill; we've been using `const string` (or const int) for everything else like this up to this point and has seemed to work great.

Reply:
It may seem that way but it certainly helps keep the load balancer type logic encapsulated in one place. The only other thing that is similar that I can think of is `EnvironmentOS`. 

The user can specify either `linux` or `windows`. But load balancer type can be `clb` or `alb` and these values are also translated to `classic` and `application` for the cli. So it makes sense to me to encapsulate this in a type rather than a const string and have supporting helper methods scattered elsewhere.

Reply:
Actually I do see your point about this being overkill. I'll remove the concept of `CLB` and `ALB` and just have `classic` and `application` as load balancer types. This would allow us to stay somewhat consistent.